### ISSUE TITLE
Cannot log in Telegram both via QR and by telephone number on Windows 10 (64bit)

### ISSUE THREAD

**Issue by AMaklakov** [telegramdesktop/tdesktop#17243]
### Steps to reproduce

I'm in Minsk/Belarus. Telegram clients for iPhones, MacBooks, and Web work perfectly fine, but when I download tdesktop version > 2.3.0 for 64 bit Windows, it does not render QR code and does not send SMS to the provided phone number.

So for now I have to use the Kotatogram client.

**Flow 1**

1. Download and install the latest version tdesktop (3.2.2) for Windows 64
2. Open it and click the "Start messaging" button

**Flow 2**

1. Download and install the latest version tdesktop (3.2.2)  for Windows 64
2. Open it and click the "Start messaging" button
3. Click the "Or log in using your phone number" button
4. Enter your phone number and proceed

### Expected behaviour

**Flow 1**

The application renders QR code.

**Flow 2**

Receive SMS from telegram.

### Actual behaviour

**Flow 1**

Spinner is shown indefinitely long time

![photo_2021-11-14 13 11 43](https://user-images.githubusercontent.com/22777424/141676757-8c4282f8-7161-4204-99a8-c9051e581161.jpeg)

**Flow 2**

No messages are received within several hours.

### Operating system

Windows 10, 64bit

### Version of Telegram Desktop

3.2.2

### Installation source

Static binary from official website

**Comment by Aokromes** [comment]
not a bug, server side limitation.

**Comment by AMaklakov** [comment]
@Aokromes thank you. But how can I overcome this issue then?

**Comment by Aokromes** [comment]
> @Aokromes thank you. But how can I overcome this issue then?

get sms on kotatogram, then login into telegram desktop with the code on kotatogram.

**Comment by john-preston** [comment]
Thanks for your report. This looks like a recent bug.

**Comment by john-preston** [comment]
@AMaklakov I'm afraid I won't be able to help without DebugLogs (type "debugmode" in Settings, accept enabling DebugLogs, relaunch the app and collect them while trying to generate QR code, then send them here).

**Comment by AMaklakov** [comment]
Reopened the app several times before figuring out how to find the proper log file. Here are my logs =)

@john-preston [log.txt](https://github.com/telegramdesktop/tdesktop/files/7534439/log.txt)

```
[2021.11.14 21:52:28] Launched version: 3002002, install beta: [FALSE], alpha: 0, debug mode: [TRUE]
[2021.11.14 21:52:28] Executable dir: C:/Users/37529/AppData/Roaming/Telegram Desktop/, name: Telegram.exe
[2021.11.14 21:52:28] Initial working dir: C:/Users/37529/AppData/Roaming/Telegram Desktop/
[2021.11.14 21:52:28] Working dir: C:/Users/37529/AppData/Roaming/Telegram Desktop/
[2021.11.14 21:52:28] Command line: C:\Users\37529\AppData\Roaming\Telegram Desktop\Telegram.exe
[2021.11.14 21:52:28] Executable path before check: C:/Users/37529/AppData/Roaming/Telegram Desktop/Telegram.exe
[2021.11.14 21:52:28] Logs started
[2021.11.14 21:52:28] Connecting local socket to Global\10e8b52ec889890c158e1d42396ecbcd-{87A94AB0-E370-4cde-98D3-ACC110C5967D}...
[2021.11.14 21:52:28] This is the only instance of Telegram, starting server and app...
[2021.11.14 21:52:28] Moved logging from 'C:/Users/37529/AppData/Roaming/Telegram Desktop/log_start1.txt' to 'C:/Users/37529/AppData/Roaming/Telegram Desktop/log.txt'!
[2021.11.14 21:52:28] Old start log 'log_start0.txt' found, deleted: [TRUE]
[2021.11.14 21:52:28] Primary screen DPI: 96
[2021.11.14 21:52:28] Font: from ':/gui/fonts/DAOpenSansRegular.ttf' loaded 'DAOpenSansRegular'
[2021.11.14 21:52:28] Font: from ':/gui/fonts/DAVazirRegular.ttf' loaded 'DAVazirRegular'
[2021.11.14 21:52:28] Font: from ':/gui/fonts/DAOpenSansRegularItalic.ttf' loaded 'DAOpenSansRegularItalic'
[2021.11.14 21:52:28] Font: from ':/gui/fonts/DAVazirRegular.ttf' loaded 'DAVazirRegular'
[2021.11.14 21:52:28] Font: from ':/gui/fonts/DAOpenSansSemiboldAsBold.ttf' loaded 'DAOpenSansSemibold'
[2021.11.14 21:52:28] Font: from ':/gui/fonts/DAVazirMediumAsBold.ttf' loaded 'DAVazirMedium'
[2021.11.14 21:52:28] Font: from ':/gui/fonts/DAOpenSansSemiboldItalicAsBold.ttf' loaded 'DAOpenSansSemiboldItalic'
[2021.11.14 21:52:28] Font: from ':/gui/fonts/DAVazirMediumAsBold.ttf' loaded 'DAVazirMedium'
[2021.11.14 21:52:28] Font: from ':/gui/fonts/DAOpenSansSemiboldAsBold.ttf' loaded 'DAOpenSansSemibold'
[2021.11.14 21:52:29] Font: from ':/gui/fonts/DAVazirMediumAsBold.ttf' loaded 'DAVazirMedium'
[2021.11.14 21:52:29] Font: from ':/gui/fonts/DAOpenSansSemiboldItalicAsBold.ttf' loaded 'DAOpenSansSemiboldItalic'
[2021.11.14 21:52:29] Font: from ':/gui/fonts/DAVazirMediumAsBold.ttf' loaded 'DAVazirMedium'
[2021.11.14 21:52:29] Using DirectX compiler 'C:\Users\37529\AppData\Roaming\Telegram Desktop\modules\x64\d3d\d3dcompiler_47.dll'.
[2021.11.14 21:52:29] App Info: reading settings...
[2021.11.14 21:52:29] App Info: reading encrypted settings...
[2021.11.14 21:52:30] OpenAL Logging Level: 3
[2021.11.14 21:52:30] Audio Playback Devices: OpenAL Soft on Динамики (High Definition Audio Device);OpenAL Soft on Digital Audio (S/PDIF) (High Definition Audio Device);OpenAL Soft on Динамики (High Definition Audio Device) #2
[2021.11.14 21:52:30] Audio Playback Default Device: OpenAL Soft on Динамики (High Definition Audio Device)
[2021.11.14 21:52:30] Audio Capture Devices: OpenAL Soft on Microphone (High Definition Audio Device)
[2021.11.14 21:52:30] Audio Capture Default Device: OpenAL Soft on Microphone (High Definition Audio Device)
[2021.11.14 21:52:30] App Info: reading accounts info...
[2021.11.14 21:52:30] App Info: reading encrypted info...
[2021.11.14 21:52:30] OpenGL: Using custom 'gpu_driver_bug_list.json'.
[2021.11.14 21:52:30] QWindowsIntegration::createPlatformOpenGLContext QSurfaceFormat(version 2.0, options QFlagsQSurfaceFormat::FormatOption(), depthBufferSize -1, redBufferSize -1, greenBufferSize -1, blueBufferSize -1, alphaBufferSize 8, stencilBufferSize -1, samples 0, swapBehavior QSurfaceFormat::DefaultSwapBehavior, swapInterval 1, colorSpace QSurfaceFormat::DefaultColorSpace, profile  QSurfaceFormat::NoProfile)
[2021.11.14 21:52:30] GPU features: QSet("disable_program_cache")
[2021.11.14 21:52:30] Disabling program cache:  GpuDescription(vendorId=0x8086, deviceId=0x416, subSysId=0x1462, revision=6, driver: "igdumdim64.dll", version=20.19.15.4531, "Intel(R) HD Graphics 4600""")
[2021.11.14 21:52:30] QWindowsOpenGLTester::supportedRenderers GpuDescription(vendorId=0x8086, deviceId=0x416, subSysId=0x1462, revision=6, driver: "igdumdim64.dll", version=20.19.15.4531, "Intel(R) HD Graphics 4600""") 2 renderer:  QFlags(0x2|0x4|0x8|0x20|0x200)
[2021.11.14 21:52:30] Qt: Using EGL from libEGL
[2021.11.14 21:52:30] Qt: Using OpenGL ES 2.0 from libGLESv2
[2021.11.14 21:52:30] QWindowsEGLStaticContext::create Created EGL display 0x1e89fbae220 v 1 . 5
[2021.11.14 21:52:31] OpenGL Profile: OpenGLES.
[2021.11.14 21:52:31] OpenGL Renderer: ANGLE (Intel, Intel(R) HD Graphics 4600 Direct3D11 vs_5_0 ps_5_0, D3D11-20.19.15.4531)
[2021.11.14 21:52:31] OpenGL Vendor: Google Inc. (Intel)
[2021.11.14 21:52:31] OpenGL Version: OpenGL ES 3.0.0 (ANGLE 2.1.7 git hash: 0bb011f9e403)
[2021.11.14 21:52:31] OpenGL Extensions: GL_ANGLE_copy_texture_3d, GL_ANGLE_texture_compression_dxt3, GL_OES_draw_elements_base_vertex, GL_KHR_parallel_shader_compile, GL_OES_draw_buffers_indexed, GL_EXT_draw_buffers_indexed, GL_EXT_draw_elements_base_vertex, GL_OES_compressed_EAC_R11_unsigned_texture, GL_EXT_blend_func_extended, GL_ANGLE_robust_client_memory, GL_EXT_debug_marker, GL_OVR_multiview2, GL_CHROMIUM_bind_uniform_location, GL_OES_compressed_EAC_R11_signed_texture, GL_ANGLE_texture_compression_dxt5, GL_EXT_sRGB, GL_EXT_texture_compression_bptc, GL_OES_surfaceless_context, GL_ANGLE_multi_draw, GL_EXT_unpack_subimage, GL_ANGLE_provoking_vertex, GL_ANGLE_multiview_multisample, GL_OES_fbo_render_mipmap, , GL_ANGLE_explicit_context_gles1, GL_EXT_disjoint_timer_query, GL_CHROMIUM_bind_generates_resource, GL_CHROMIUM_copy_texture, GL_OES_mapbuffer, GL_EXT_read_format_bgra, GL_EXT_texture_compression_rgtc, GL_OES_get_program_binary, GL_ANGLE_instanced_arrays, GL_OES_rgb8_rgba8, GL_ANGLE_translated_shader_source, GL_CHROMIUM_sync_query, GL_EXT_texture_compression_s3tc_srgb, GL_EXT_texture_norm16, GL_OES_texture_half_float_linear, GL_EXT_multisampled_render_to_texture, GL_EXT_occlusion_query_boolean, GL_OES_element_index_uint, GL_ANGLE_framebuffer_multisample, GL_EXT_texture_storage, GL_OES_texture_float_linear, GL_ANGLE_request_extension, GL_EXT_blend_minmax, GL_ANGLE_get_serialized_context_string, GL_ANGLE_pack_reverse_row_order, GL_EXT_color_buffer_half_float, GL_OES_texture_stencil8, GL_OES_EGL_image_external, GL_ANGLE_program_cache_control, GL_EXT_discard_framebuffer, GL_OES_texture_float, GL_EXT_instanced_arrays, GL_EXT_shader_texture_lod, GL_CHROMIUM_copy_compressed_texture, GL_EXT_EGL_image_external_wrap_modes, GL_ANGLE_memory_size, GL_OES_compressed_EAC_RG11_unsigned_texture, GL_OES_compressed_ETC2_RGBA8_texture, GL_KHR_debug, GL_OES_compressed_ETC2_punchthroughA_RGBA8_texture, GL_EXT_texture_format_BGRA8888, GL_ANGLE_client_arrays, GL_OES_compressed_ETC2_sRGB8_alpha8_texture, GL_NV_pack_subimage, GL_OVR_multiview, GL_WEBGL_video_texture, GL_ANGLE_lossy_etc_decode, GL_OES_EGL_image_external_essl3, GL_OES_texture_npot, GL_ANGLE_framebuffer_blit, GL_EXT_color_buffer_float, GL_NV_fence, GL_ANGLE_texture_usage, GL_CHROMIUM_lose_context, GL_OES_EGL_image, GL_EXT_float_blend, GL_EXT_texture_filter_anisotropic, GL_OES_compressed_EAC_RG11_signed_texture, GL_OES_depth32, GL_ANGLE_explicit_context, GL_OES_compressed_ETC2_sRGB8_texture, GL_EXT_frag_depth, GL_OES_compressed_ETC2_punchthroughA_sRGB8_alpha_texture, GL_EXT_texture_rg, GL_OES_vertex_array_object, GL_ANGLE_depth_texture, GL_EXT_texture_compression_dxt1, GL_EXT_draw_buffers, GL_OES_packed_depth_stencil, GL_OES_standard_derivatives, GL_ANGLE_get_tex_level_parameter, GL_ANGLE_base_vertex_base_instance, GL_EXT_robustness, GL_EXT_map_buffer_range, GL_ANGLE_texture_multisample, GL_OES_compressed_ETC2_RGB8_texture, GL_OES_texture_half_float, GL_NV_EGL_stream_consumer_external, GL_EXT_debug_label, GL_OES_depth24, GL_OES_texture_border_clamp, GL_NV_pixel_buffer_object
[2021.11.14 21:52:31] EGL Extensions: EGL_EXT_create_context_robustness, EGL_ANGLE_d3d_share_handle_client_buffer, EGL_ANGLE_d3d_texture_client_buffer, EGL_ANGLE_surface_d3d_texture_2d_share_handle, EGL_ANGLE_query_surface_pointer, EGL_ANGLE_window_fixed_size, EGL_ANGLE_keyed_mutex, EGL_ANGLE_surface_orientation, EGL_ANGLE_direct_composition, EGL_ANGLE_windows_ui_composition, EGL_NV_post_sub_buffer, EGL_KHR_create_context, EGL_KHR_image, EGL_KHR_image_base, EGL_KHR_gl_texture_2D_image, EGL_KHR_gl_texture_cubemap_image, EGL_KHR_gl_renderbuffer_image, EGL_KHR_get_all_proc_addresses, EGL_KHR_stream, EGL_KHR_stream_consumer_gltexture, EGL_NV_stream_consumer_gltexture_yuv, EGL_ANGLE_flexible_surface_compatibility, EGL_ANGLE_stream_producer_d3d_texture, EGL_ANGLE_create_context_webgl_compatibility, EGL_CHROMIUM_create_context_bind_generates_resource, EGL_CHROMIUM_sync_control, EGL_EXT_pixel_format_float, EGL_KHR_surfaceless_context, EGL_ANGLE_display_texture_share_group, EGL_ANGLE_display_semaphore_share_group, EGL_ANGLE_create_context_client_arrays, EGL_ANGLE_program_cache_control, EGL_ANGLE_robust_resource_initialization, EGL_ANGLE_create_context_extensions_enabled, EGL_ANDROID_blob_cache, EGL_ANDROID_recordable, EGL_ANGLE_image_d3d11_texture, EGL_ANGLE_create_context_backwards_compatible, EGL_KHR_create_context_no_error, EGL_KHR_reusable_sync,
[2021.11.14 21:52:31] OpenGL: QOpenGLContext created, version: 3.3.
[2021.11.14 21:52:31] OpenGL: [TRUE] (OverlayWidget)
[2021.11.14 21:52:31] QWindowsIntegration::createPlatformOpenGLContext QSurfaceFormat(version 2.0, options QFlagsQSurfaceFormat::FormatOption(), depthBufferSize -1, redBufferSize -1, greenBufferSize -1, blueBufferSize -1, alphaBufferSize 8, stencilBufferSize -1, samples -1, swapBehavior QSurfaceFormat::DefaultSwapBehavior, swapInterval 1, colorSpace QSurfaceFormat::DefaultColorSpace, profile  QSurfaceFormat::NoProfile)
[2021.11.14 21:52:31] QWindowsIntegration::createPlatformOpenGLContext QSurfaceFormat(version 2.0, options QFlagsQSurfaceFormat::FormatOption(), depthBufferSize -1, redBufferSize -1, greenBufferSize -1, blueBufferSize -1, alphaBufferSize -1, stencilBufferSize -1, samples 0, swapBehavior QSurfaceFormat::DefaultSwapBehavior, swapInterval 1, colorSpace QSurfaceFormat::DefaultColorSpace, profile  QSurfaceFormat::NoProfile)
[2021.11.14 21:52:32] RPC Error: request 1 got fail with code 400, error CONNECTION_DEVICE_MODEL_EMPTY
[2021.11.14 21:52:32] MTP Error: failed to get config!
[2021.11.14 21:52:32] RPC Error: request 2 got fail with code 400, error CONNECTION_DEVICE_MODEL_EMPTY
[2021.11.14 21:52:32] RPC Error: request 3 got fail with code 400, error CONNECTION_DEVICE_MODEL_EMPTY
[2021.11.14 21:52:32] RPC Error: request 4 got fail with code 400, error CONNECTION_DEVICE_MODEL_EMPTY
[2021.11.14 21:52:32] API Error: getting countries failed with error CONNECTION_DEVICE_MODEL_EMPTY
[2021.11.14 21:52:38] QPainter::begin: Paint device returned engine == 0, type: 3
[2021.11.14 21:52:38] QWidget::render: Cannot render with an inactive painter
[2021.11.14 21:52:38] QPainter::begin: Paint device returned engine == 0, type: 3
[2021.11.14 21:52:38] QWidget::render: Cannot render with an inactive painter
[2021.11.14 21:52:38] RPC Error: request 6 got fail with code 400, error CONNECTION_DEVICE_MODEL_EMPTY
[2021.11.14 21:52:39] Config Error: Failed to parse dns response JSON, error: illegal value
[2021.11.14 21:52:39] Config Error: Bad data size 0 required 344
[2021.11.14 21:52:42] RPC Error: request 7 got fail with code 400, error CONNECTION_DEVICE_MODEL_EMPTY
[2021.11.14 21:52:42] MTP Error: failed to get config!
[2021.11.14 21:52:42] RPC Error: request 5 got fail with code 400, error CONNECTION_DEVICE_MODEL_EMPTY
[2021.11.14 21:52:42] MTP Error: failed to get config!
[2021.11.14 21:52:46] Config Error: Failed to parse dns response JSON, error: illegal value
[2021.11.14 21:52:46] Config Error: Bad data size 0 required 344
[2021.11.14 21:52:46] Config Error: Failed to parse dns response JSON, error: illegal value
[2021.11.14 21:52:46] Config Error: Bad data size 0 required 344
[2021.11.14 21:52:46] RPC Error: request 8 got fail with code 400, error CONNECTION_DEVICE_MODEL_EMPTY
[2021.11.14 21:52:46] MTP Error: failed to get config!
[2021.11.14 21:52:47] RPC Error: request 9 got fail with code 400, error CONNECTION_DEVICE_MODEL_EMPTY
[2021.11.14 21:52:47] MTP Error: failed to get config!
[2021.11.14 21:52:54] RPC Error: request 10 got fail with code 400, error CONNECTION_DEVICE_MODEL_EMPTY
[2021.11.14 21:52:54] MTP Error: failed to get config!
[2021.11.14 21:52:58] RPC Error: request 11 got fail with code 400, error CONNECTION_DEVICE_MODEL_EMPTY
[2021.11.14 21:52:58] MTP Error: failed to get config!
[2021.11.14 21:53:00] RPC Error: request 12 got fail with code 400, error CONNECTION_DEVICE_MODEL_EMPTY
[2021.11.14 21:53:00] MTP Error: failed to get config!
[2021.11.14 21:53:06] RPC Error: request 13 got fail with code 400, error CONNECTION_DEVICE_MODEL_EMPTY
[2021.11.14 21:53:06] MTP Error: failed to get config!
[2021.11.14 21:53:06] RPC Error: request 14 got fail with code 400, error CONNECTION_DEVICE_MODEL_EMPTY
[2021.11.14 21:53:06] MTP Error: failed to get config!
[2021.11.14 21:53:11] RPC Error: request 15 got fail with code 400, error CONNECTION_DEVICE_MODEL_EMPTY
[2021.11.14 21:53:11] MTP Error: failed to get config!
[2021.11.14 21:53:12] RPC Error: request 16 got fail with code 400, error CONNECTION_DEVICE_MODEL_EMPTY
[2021.11.14 21:53:12] MTP Error: failed to get config!
[2021.11.14 21:53:18] RPC Error: request 17 got fail with code 400, error CONNECTION_DEVICE_MODEL_EMPTY
[2021.11.14 21:53:18] MTP Error: failed to get config!
[2021.11.14 21:53:18] Config Error: Failed to parse dns response JSON, error: illegal value
[2021.11.14 21:53:18] Config Error: Bad data size 0 required 344
[2021.11.14 21:53:19] RPC Error: request 18 got fail with code 400, error CONNECTION_DEVICE_MODEL_EMPTY
[2021.11.14 21:53:19] MTP Error: failed to get config!
[2021.11.14 21:53:24] RPC Error: request 19 got fail with code 400, error CONNECTION_DEVICE_MODEL_EMPTY
[2021.11.14 21:53:24] MTP Error: failed to get config!
[2021.11.14 21:53:26] RPC Error: request 20 got fail with code 400, error CONNECTION_DEVICE_MODEL_EMPTY
[2021.11.14 21:53:26] MTP Error: failed to get config!
[2021.11.14 21:53:33] RPC Error: request 21 got fail with code 400, error CONNECTION_DEVICE_MODEL_EMPTY
[2021.11.14 21:53:33] MTP Error: failed to get config!
[2021.11.14 21:53:34] Config Error: Failed to parse dns response JSON, error: illegal value
[2021.11.14 21:53:34] Config Error: Bad data size 0 required 344
[2021.11.14 21:53:35] RPC Error: request 22 got fail with code 400, error CONNECTION_DEVICE_MODEL_EMPTY
[2021.11.14 21:53:35] MTP Error: failed to get config!
[2021.11.14 21:53:36] RPC Error: request 23 got fail with code 400, error CONNECTION_DEVICE_MODEL_EMPTY
[2021.11.14 21:53:36] MTP Error: failed to get config!
[2021.11.14 21:53:43] RPC Error: request 25 got fail with code 400, error CONNECTION_DEVICE_MODEL_EMPTY
[2021.11.14 21:53:43] MTP Error: failed to get config!
[2021.11.14 21:53:43] RPC Error: request 24 got fail with code 400, error CONNECTION_DEVICE_MODEL_EMPTY
[2021.11.14 21:53:43] MTP Error: failed to get config!
[2021.11.14 21:53:49] Config Error: Failed to parse dns response JSON, error: illegal value
[2021.11.14 21:53:49] Config Error: Bad data size 0 required 344
[2021.11.14 21:53:50] RPC Error: request 26 got fail with code 400, error CONNECTION_DEVICE_MODEL_EMPTY
[2021.11.14 21:53:50] MTP Error: failed to get config!
[2021.11.14 21:53:50] RPC Error: request 27 got fail with code 400, error CONNECTION_DEVICE_MODEL_EMPTY
[2021.11.14 21:53:50] MTP Error: failed to get config!
[2021.11.14 21:53:56] RPC Error: request 28 got fail with code 400, error CONNECTION_DEVICE_MODEL_EMPTY
[2021.11.14 21:53:56] MTP Error: failed to get config!
[2021.11.14 21:53:58] Config Error: Failed to parse dns response JSON, error: illegal value
[2021.11.14 21:53:58] Config Error: Bad data size 0 required 344
[2021.11.14 21:53:59] RPC Error: request 29 got fail with code 400, error CONNECTION_DEVICE_MODEL_EMPTY
[2021.11.14 21:53:59] MTP Error: failed to get config!
[2021.11.14 21:54:02] RPC Error: request 30 got fail with code 400, error CONNECTION_DEVICE_MODEL_EMPTY
[2021.11.14 21:54:02] MTP Error: failed to get config!
[2021.11.14 21:54:06] RPC Error: request 31 got fail with code 400, error CONNECTION_DEVICE_MODEL_EMPTY
[2021.11.14 21:54:06] MTP Error: failed to get config!
[2021.11.14 21:54:08] RPC Error: request 32 got fail with code 400, error CONNECTION_DEVICE_MODEL_EMPTY
[2021.11.14 21:54:08] MTP Error: failed to get config!
[2021.11.14 21:54:14] RPC Error: request 33 got fail with code 400, error CONNECTION_DEVICE_MODEL_EMPTY
[2021.11.14 21:54:14] MTP Error: failed to get config!
[2021.11.14 21:54:15] RPC Error: request 34 got fail with code 400, error CONNECTION_DEVICE_MODEL_EMPTY
[2021.11.14 21:54:15] MTP Error: failed to get config!
[2021.11.14 21:54:20] RPC Error: request 35 got fail with code 400, error CONNECTION_DEVICE_MODEL_EMPTY
[2021.11.14 21:54:20] MTP Error: failed to get config!
[2021.11.14 21:54:23] RPC Error: request 36 got fail with code 400, error CONNECTION_DEVICE_MODEL_EMPTY
[2021.11.14 21:54:23] MTP Error: failed to get config!
[2021.11.14 21:54:26] RPC Error: request 37 got fail with code 400, error CONNECTION_DEVICE_MODEL_EMPTY
[2021.11.14 21:54:26] MTP Error: failed to get config!
[2021.11.14 21:54:31] Config Error: Failed to parse dns response JSON, error: illegal value
[2021.11.14 21:54:31] Config Error: Bad data size 0 required 344
[2021.11.14 21:54:31] RPC Error: request 38 got fail with code 400, error CONNECTION_DEVICE_MODEL_EMPTY
[2021.11.14 21:54:31] MTP Error: failed to get config!
[2021.11.14 21:54:32] RPC Error: request 39 got fail with code 400, error CONNECTION_DEVICE_MODEL_EMPTY
[2021.11.14 21:54:32] MTP Error: failed to get config!
[2021.11.14 21:54:39] RPC Error: request 41 got fail with code 400, error CONNECTION_DEVICE_MODEL_EMPTY
[2021.11.14 21:54:39] MTP Error: failed to get config!
[2021.11.14 21:54:39] RPC Error: request 40 got fail with code 400, error CONNECTION_DEVICE_MODEL_EMPTY
[2021.11.14 21:54:39] MTP Error: failed to get config!
[2021.11.14 21:54:45] Config Error: Failed to parse dns response JSON, error: illegal value
[2021.11.14 21:54:45] Config Error: Bad data size 0 required 344
[2021.11.14 21:54:46] RPC Error: request 42 got fail with code 400, error CONNECTION_DEVICE_MODEL_EMPTY
[2021.11.14 21:54:46] MTP Error: failed to get config!
[2021.11.14 21:54:46] Config Error: Failed to parse dns response JSON, error: illegal value
[2021.11.14 21:54:46] Config Error: Bad data size 0 required 344
[2021.11.14 21:54:46] RPC Error: request 43 got fail with code 400, error CONNECTION_DEVICE_MODEL_EMPTY
[2021.11.14 21:54:46] MTP Error: failed to get config!
```

**Comment by john-preston** [comment]
@AMaklakov I need "mtp_...txt" log from a "DebugLogs" folder near Telegram.exe file.

**Comment by AMaklakov** [comment]
@john-preston here you are:

[mtp_21_45.txt](https://github.com/telegramdesktop/tdesktop/files/7536627/mtp_21_45.txt)

**Comment by john-preston** [comment]
@AMaklakov Thanks. Somehow a single symbol with code '4' (non-printable) got in the device model, it is trimmed by the server, but left as is by the app.

**Comment by AMaklakov** [comment]
@john-preston Thx!

**Comment by john-preston** [comment]
@AMaklakov Please confirm that it is fixed in the version 3.2.5.

**Comment by AMaklakov** [comment]
@john-preston Thx! It has been fixed! I will find the person who named the device model like this =)
